src/globals/scss/settings/_spacing.scss

```diff
@@ -1,16 +1,67 @@
-// Spacing scale
-$govuk-spacing-scale-0: 0;
-$govuk-spacing-scale-1: 5px;
-$govuk-spacing-scale-2: 10px;
-$govuk-spacing-scale-3: 15px;
-$govuk-spacing-scale-4: 20px;
-$govuk-spacing-scale-5: 30px;
-$govuk-spacing-scale-6: 40px;
-$govuk-spacing-scale-7: 50px;
-$govuk-spacing-scale-8: 60px;
+// Responsive spacing maps
+//
+// These definitions are used to generate responsive spacing that adapts
+// according to the breakpoints (see 'helpers/spacing').
+
+// You can define different behaviour on tablet and desktop. The 'null'
+// breakpoint is for mobile.
+
+// To use the points on the spacing scale:
+//  Combine them with 'govuk-spacing-unit' or 'govuk-spacing-responsive' mixins
+// (see 'helpers/spacing').
+
+$govuk-spacing-scale-0: (
+  null: 0,
+  tablet: 0
```

Comment:
These could be omitted for 0 thru 3? Avoids generating media queries that don't do anything.

Reply:
Although… that would break the `govuk-spacing-unit` function… 🤔

Reply:
Might you want to use a media query with zero padding/margin to override existing styling to be zero on all breakpoints? It's a weird use case. I've taken out the zero declaration from the base scale but left it in the responsive scale.

Reply:
Ooh, interesting – I hadn't thought of that. I would imagine that because the overrides use `!important` they'd trump even media queries, but I don't actually know what the behaviour would be.